gromacs/scaling.py

```diff
@@ -211,16 +219,17 @@ def partial_tempering(args):
                 impropertypes[name].append(it)
         print("Build impropertypes dictionary with {0} entries".format(len(impropertypes)))
 
-        if 'Protein' in top.dict_molname_mol:
-                mol = top.dict_molname_mol['Protein']
-                for at in mol.atoms: at.charge *= math.sqrt(args.scale_protein)
-                mol = scale_dihedrals(mol, dihedraltypes, args.scale_protein, banned_lines)
-                mol = scale_impropers(mol, impropertypes, 1.0, banned_lines)
+        for molname_mol in top.dict_molname_mol.keys():
+            if not 'Protein' in molname_mol: continue
```

Comment:
I prefer putting `continue` on a separate line (as well as having explicit `for` blocks).